common/soroban-rpc/bin/start

```diff
@@ -3,11 +3,6 @@
 set -e
 set -o pipefail
 
-while ! (curl -sf http://localhost:8001/ | jq --exit-status '.ingest_latest_ledger > 1'); do
-  echo "Waiting for horizon to be available..."
-  sleep 1
-done
-
```

Comment:
it was duplicative since start script was doing the same, i put it back in, it's not a big deal to have the check done in both places, want to keep it in start in any case as it will log out horizon ingest status regardless of soroban-rpc modes.